ddpui/services/org_cleanup_service.py

```
@@ -0,0 +1,356 @@
+
+        # clear up github PAT from everywhere if exists
+        orgdbt = self.org.dbt
+        for secret_block in OrgPrefectBlockv1.objects.filter(org=self.org, block_type=SECRET).all():
+            logger.info(f"will delete secret block {secret_block.block_name} from prefect & DB")
+            logger.info("will also delete github PAT if exists in secrets manager")
+            if not self.dry_run:
+                try:
+                    prefect_service.delete_secret_block(secret_block.block_id)
+                    logger.info(f"deleted secret block {secret_block.block_name} from prefect")
+                except Exception:  # pylint:disable=broad-exception-caught
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)